source/small/string.h

```diff
@@ -2,8 +2,8 @@
 // Created by Alan Freitas on 8/1/21.
 //
 
-#ifndef SMALL_UTF8_STRING_H
-#define SMALL_UTF8_STRING_H
+#ifndef SMALL_STRING_H8_STRING_H
```

Comment:
This one didn't work that well. It should probably be `SMALL_STRING_H`.

It could be a bug in the linter (because of the digit in `SMALL_UTF8_STRING_H`?) or we just need to run it again for some reason.

Reply:
Yes,  this one didn't work well.  I'll try to run again the linter. We'll probably need to open an issue if it doesn't work, right?

Reply:
Yeah, this is probably a bug in the linter. The linter is considering the guard just until the number, as you can see in the log:

``Convert guard from SMALL_UTF to SMALL_STRING_H``

Reply:
This is easy to fix and easy to "fix without fixing". 

We can fix it simply by allowing numbers in the macro expression: https://github.com/marcosfpr/small/blob/c4e64a5f3114d610524342914c6c18e262176c27/dev-tools/linter/main.cpp#L213

Or we cannot fix it and still get the right guard by just renaming it to anything without a number and running the linter again.

At some point, it might be worth moving this linter to another repo.

Reply:
And yes. In the second case, we could open an issue.

Reply:
Make sense. In that particular case, I can rename the header and open an issue to fix that. Also, I agree with you that probably this should be in another repository.